sdk/compute/Microsoft.Azure.Management.Compute/tests/VMScaleSetTests/VMScaleSetTestsBase.cs

```diff
@@ -283,6 +283,7 @@ protected VirtualMachineScaleSetExtension GetAzureDiskEncryptionExtension()
             string dedicatedHostGroupReferenceId = null,
             string dedicatedHostGroupName = null,
             string dedicatedHostName = null,
+            string capacityReservationGroupReferenceId = null,
```

Comment:
could you add after userData? next line. 
To make sure past uses of this function stays consistent